code/src/App.js

```diff
@@ -1,9 +1,70 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Header } from './components/Header.js'
+import { Summary } from './components/Summary.js'
+import "./app.css"
 
 export const App = () => {
+
+  const [showSummary, setShowSummary] = useState(false)
+  const howManyPlants = ['0', '1-3', '3-6', '6-10', '10-20', '20 +']
+  const [numberOfPlants, setNumberOfPlants] = useState('')
+  const [livingPlace, setLivingPlace] = useState('')
+  const [favouritePlant, setFavouritePlant] = useState('')
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    setShowSummary(true)
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <div class='app'>
+      <Header />
+      {!showSummary ? (
+        <form onSubmit={handleSubmit} className='box form'>
+          <div>
+            <label>
+              <h3>How do you live?</h3>
+              <select
+                onChange={event => setLivingPlace(event.target.value)}
+                value={livingPlace}
+                required>
+                <option value=''>Select housing</option>
```

Comment:
Guess you don't want this to be a noption
```suggestion
                <option value='' disabled>Select housing</option>
```